src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -408,4 +408,12 @@ public boolean scanPlayer(int row, int col ) { // Shouldn't the battleship and s
             return false;
         }
     }
+
+    public void setEzmode(int i){
+        if(i == 0){
+            ezmode = true;
```

Comment:
We will have to move the call to the easy and hard placement of AI ships into here once Jiayu gets his finished